src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Channel.xml

```diff
@@ -25,6 +25,14 @@
                 </attribute>
             </collectionOperation>
 
+            <collectionOperation name="shop_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/shop/channels</attribute>
+                <attribute name="normalization_context">
+                    <attribute name="groups">shop:channels:read</attribute>
```

Comment:
```suggestion
                    <attribute name="groups">shop:channel:read</attribute>
```

Reply:
if I use the same name I will then serialise another 2 fields to different endpoint

Reply:
Oh, I see now ;d
If 2 extra fields in a different endpoint is a problem (maybe it's not a problem), I think it will be better to set `shop:channel:read` in the normalization context and add a second group only for these 2 fields.

Reply:
are these extra fields a problem? probably not (but I don't know which responses will be changed), ALL serialization groups are consistent, so we shouldn't make an exception for it

Reply:
Ok, If I understand the issue correctly it is related to a lack of distinguishing between item and collection serialization. Idea was to split read serialization into two groups:
```xml
                    <attribute name="groups">shop:channel:read:list</attribute>
```
and
```xml
                    <attribute name="groups">shop:channel:read:item</attribute>
```

but for now, I would say that it is not a big deal to expose two additional fields (on the other hand reduction of them would be bc break). But split of serialization groups will be separate effort

Reply:
To make it clear it's gonna be `defaultLocale`, `locales` and `currencies` additional fields. I decided to leave the same group name and serialise them anyway.